packages/flutter/lib/src/gestures/binding.dart

```diff
@@ -389,14 +388,14 @@ mixin GestureBinding on BindingBase implements HitTestable, HitTestDispatcher, H
 
   void _handleSampleTimeChanged() {
     if (!locked) {
-      _flushPointerEventQueue();
+      _resampler.sample(samplingOffset);
```

Comment:
What if resamplingEnabled is false here? Should we stop the resampler instead then?

Reply:
This can happen when resample was on but is turned off, and the logic should be to just discard the remaining events queued in the resampler? @dreveman

Reply:
Updated. PTAL.